manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml

```diff
@@ -230,6 +230,12 @@ rules:
       - get
       - list
       - delete
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
```

Comment:
It would be useful to add a comment with short description on why this permission is needed.

Reply:
That's a good point! will do